content/_index.md

```diff
@@ -17,106 +17,143 @@ g3Teaser:
+    title: Customize or Develop Your Gen3 Experience
+    detail: The Gen3 services are modular and data resources can choose which features to include. The Gen3 framework services expose powerful Application Programming Interfaces (APIs) that empower interoperation of a healthy data ecosystem and can provide the foundation for developing new tools for sharing and analyzing data across resources in the cloud.
+  f5:
+    title: Open-Source Community
+    detail: We believe in open-source software drives better software development. Engage with our Gen3’s user community, develop new features, ask a question to our Support Team, or discuss with other scholars about similar technical or scientific questions during their experience with Gen3.
+  f6:
+    title: Powerful APIs
+    detail: Each service exposes open Application Programing Interfaces (APIs) for data submission, indexing data objects, associating metadata with the data objects, sending custom query requests, or enabling programmatic data search and custom app development.
+
 g3Roles:
   title: Gen3 helps you as...
   r1:
     name: Bioinformaticians
     fig: /figs/roles/icon-bio.svg
-    detail: Data resources using Gen3 provide open APIs for accessing organized data with persistent digital identifiers to allow you to focus on creating unique pipelines or deep learning algorithms for your analyses.
+    detail: Create unique pipelines or deep learning algorithms for your analyses using Gen3 workflow execution services and open APIs for accessing organized data with persistent digital identifiers.
     button:
       caption: Pipeline Library
       target: _blank
       link: https://github.com/uc-cdis/ndh-demo
   r2:
     name: Researchers
     fig: /figs/roles/icon-research.svg
-    detail: Use our smart dashboard to explore and find data from multiple sources to meet the needs of your research. Collaborate with others in your field to combine data for powerful, cross-project meta-analyses.
+    detail: Use our smart dashboard to explore and find data from multiple sources to advance your research. Collaborate with others in your field to combine data for powerful, cross-project meta-analyses.
     button:
       caption: Gen3 User Guide
       target: _self
       link: /resources/user
   r3:
     name: Developers
     fig: /figs/roles/icon-devs.svg
-    detail: Integrate Gen3 open APIs into your services and applications, and leverage our open-source software libraries to expedite development of your own tools. Develop custom apps that can be marketed to a variety of data resources that utilize a common set of framework services and APIs.
+    detail: Integrate Gen3 open APIs into your services and applications, and leverage our open-source software libraries to development of custom apps.
     button:
       caption: Gen3 Developer
       target: _self
       link: /resources/developer
+
+g3User1:
```

Comment:
I would rather not call this `g3User1 `, either put it into `g3User` block below as `quote2` `by2`, or give it a more meaningful name

Reply:
ha oke, will call is `g3RLG` for the initials of the head.